packages/elements-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-utils",
-  "version": "7.1.2",
+  "version": "7.2.0",
```

Comment:
Did something change with elements-utils?

Reply:
I don't know. I assumed @mnaumanali94 wanted minor versions for all?
https://stoplight-internal.slack.com/archives/C029YQEEALD/p1638903820468300

Reply:
Reverted back